src/sentry/models/groupsearchviewlastseen.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, region_silo_model
+from sentry.db.models.base import Model
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@region_silo_model
+class GroupSearchViewLastSeen(Model):
```

Comment:
I'd recommend using `DefaultFieldsModel` so that you get `date_added` and `date_updated` as well.